pkg/eventhandler/eventhandlers.go

```diff
@@ -304,7 +286,7 @@ func sendJobFailedEvent(myKeptn *keptnv2.Keptn, jobName string, serviceName stri
 	}
 }
 
-func sendTaskFinishedEvent(myKeptn *keptnv2.Keptn, serviceName string, jobLogs []jobLogs, data dataForFinishedEvent) {
+func getTaskFinishedEvent(event sdk.KeptnEvent, receivedEventData keptn.EventProperties, jobLogs []jobLogs, data dataForFinishedEvent) interface{} {
```

Comment:
I don't think that the name of this function is suitable, as it computes either a `keptnv2.TestFinishedEventData` or `&keptnv2.EventData` event and returns an `interface{}`, maybe this function should be refactored a little bit to make use of some type in the return value.

Reply:
I'm not sure if this is needed since we only use the function in combination with the `go-sdk`, which also uses `interface{}`, but I'm open to suggestions.

Reply:
If that's the case, please add this as comment to the function header that explains the need for using the `interface{}` type. At least that way it's properly documented and avoid confusion in the future.